app/helpers/application_helper.rb

```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+  #for nice notifications with toastr
+  def custom_bootstrap_flash
+    flash_messages = []
+    flash.each do |type, message|
+      type = 'success' if type == 'notice'
+      type = 'error'   if type == 'alert'
+      puts type
+
+      text = "<script>toastr.#{type}('#{message}');</script>"
```

Comment:
The second part of the helper module should sanitize any xss vulns by using .html_safe right?

```rails
     text = "<script>toastr.#{type}('#{message}');</script>"
      flash_messages << text.html_safe if message
    end
    flash_messages.join("\n").html_safe
```